cloudferry_devlab/cloudferry_devlab/generate_load.py

```diff
@@ -943,6 +943,7 @@ def delete_image_on_dst(self):
         for image in images_to_delete:
             image_id = self.dst_cloud.get_image_id(image['name'])
             self.dst_cloud.glanceclient.images.delete(image_id)
+            self.dst_cloud.clean_tools.delete_image_by_id_from_db(image_id)
```

Comment:
This line needs to be deleted, because image here is deleted on purpose and should only be done via API. I suspect this change results in functional test failure we see:

```
The image with name deleted_on_dst have the same ID on dst - must be different for this image,because this image was migrated and deleted on dst. On the next migration must be generated new ID
>>  raise self.failureException('The image with name deleted_on_dst have the same ID on dst - must be different for this image,because this image was migrated and deleted on dst. On the next migration must be generated new ID')
```